molecule.yml

```diff
@@ -2,6 +2,11 @@
 ansible:
   verbose:
   sudo: True
+  host_vars:
```

Comment:
I think that using `host_vars` here keeps us having to manage different combinations of `pip_version` across supported platforms.

If we use `group_vars` instead, we can pull from it blindly no matter what platform:

```yaml
group_vars:
  all:
    - pip_version: "*"
```